src/main/scala/net/ceedubs/ficus/readers/ValueReader.scala

```diff
@@ -8,6 +8,8 @@ trait ValueReader[A] { self =>
   /** Reads the value at the path `path` in the Config */
   def read(config: Config, path: String): A
 
+  def read(config : Config) : A = throw new Exception("bla") //TODO: FIX THIS EXCEPTION
```

Comment:
The difficulty of deciding what to do here is an indicator that it doesn't belong to this trait. A new trait (possibly inheriting this one) makes sense.